### DISCUSSION TITLE
Remove Code Block Line Wrapping

### DISCUSSION THREAD

**Discussion by GraphicOwls** [notaapp/talk#354] [Feature Requests]
I noticed in this latest update that all code blocks have line wrapping now. I am personally not a fan of this other than if the language is just plain text. Can we get an option to turn off line-wrapping?

**Comment by marlonjames71** [comment]
Agreed. Makes code harder to read.

**Comment by astoilkov** [comment]
Hmm, we haven't turned off line wrapping. Code block lines have always wrapped. 

Actually, line wrapping is hard to implement for us. This is why we still haven't done it.

**Reply by marlonjames71** [reply]
Wait, I'm confused. You're saying you haven't turned it off and that it has always wrapped. But, the next line says line wrapping is hard and that's why you haven't done it.

Is line wrapping supposed to be there or is it not supposed to be there?

**Reply by GraphicOwls** [reply]
I think he's saying they haven't turned it off because it was never a like that to begin with. Also, that it's hard for them to implement which is mainly why they haven't done it.

**Reply by marlonjames71** [reply]
Ahh okay

**Reply by astoilkov** [reply]
Yeah, this is what I meant, sorry. I actually meant that removing line wrapping is hard.

**Reply by marlonjames71** [reply]
Gotcha. Thanks for the clarification.
